test/test_utils/__init__.py

```diff
@@ -100,6 +100,9 @@
 
 SAGEMAKER_EXECUTION_REGIONS = ["us-west-2", "us-east-1", "eu-west-1"]
 
+UPGRADE_ECR_REPO_NAME = f"""upgraded-image-repo-{boto3.client("sts").get_caller_identity().get("Account")}"""
+ECR_SCAN_HELPER_BUCKET = f"""ecr-scan-helper-{boto3.client("sts").get_caller_identity().get("Account")}"""
```

Comment:
Please use `boto3.client("sts", region_name=test_utils.DEFAULT_REGION)` in order to avoid using the global STS endpoint.